libcodechecker/libclient/thrift_helper.py

```diff
@@ -122,6 +122,20 @@ def getCheckerCounts(self, base_run_ids, reportFilter, cmpData, limit,
                          offset):
         pass
 
+    # SOURCE COMPONENT RELATED API CALLS
+
+    @ThriftClientCall
+    def addSourceComponent(self, name, value, description):
+        pass
+
+    @ThriftClientCall
+    def getSourceComponents(self, component_filter):
+        pass
+
+    @ThriftClientCall
+    def removeSourceComponent(self, name):
```

Comment:
Will we support to remove them using a filter or multiple names?

Reply:
I think it is better to remove only one component at the same time. This problem similar to #1362.